extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -632,3 +633,158 @@ descendant_tag_list: {
   tag: "VAR"
   tag: "WBR"
 }
+tags: {  # <amp-story-page-attachment>
+  html_format: AMP
+  tag_name: "AMP-STORY-PAGE-ATTACHMENT"
+  mandatory_ancestor: "AMP-STORY-PAGE"
+  descendant_tag_list: "amp-story-page-attachment-allowed-descendants"
+  mandatory_last_child: true
+  attrs: {
+    name: "layout"
+    mandatory: true
+    value: "nodisplay"
+  }
+}
+descendant_tag_list {
+  name: "amp-story-page-attachment-allowed-descendants"
+  tag: "A"
+  tag: "ABBR"
+  tag: "ADDRESS"
+  tag: "AMP-3D-GLTF"
+  tag: "AMP-3Q-PLAYER"
+  tag: "AMP-ACCORDION"
+  tag: "AMP-AUDIO"
+  tag: "AMP-BEOPINION"
+  tag: "AMP-BODYMOVIN-ANIMATION"
+  tag: "AMP-BRID-PLAYER"
+  tag: "AMP-BRIGHTCOVE"
+  tag: "AMP-BYSIDE-CONTENT"
+  tag: "AMP-CALL-TRACKING"
+  tag: "AMP-CAROUSEL"
+  tag: "AMP-DAILYMOTION"
+  tag: "AMP-DATE-COUNTDOWN"
+  tag: "AMP-EMBEDLY-CARD"
+  tag: "AMP-FACEBOOK"
+  tag: "AMP-FACEBOOK-COMMENTS"
+  tag: "AMP-FACEBOOK-LIKE"
+  tag: "AMP-FACEBOOK-PAGE"
+  tag: "AMP-FIT-TEXT"
+  tag: "AMP-FX-COLLECTION"
+  tag: "AMP-FX-FLYING-CARPET"
+  tag: "AMP-GFYCAT"
+  tag: "AMP-GFYCAT"
+  tag: "AMP-GIST"
+  tag: "AMP-GIST"
+  tag: "AMP-GOOGLE-DOCUMENT-EMBED"
+  tag: "AMP-GOOGLE-VRVIEW-IMAGE"
+  tag: "AMP-GOOGLE-VRVIEW-IMAGE"
+  tag: "AMP-HULU"
+  tag: "AMP-IMA-VIDEO"
+  tag: "AMP-IMAGE-SLIDER"
+  tag: "AMP-IMG"
+  tag: "AMP-IMGUR"
+  tag: "AMP-INSTAGRAM"
+  tag: "AMP-IZLESENE"
+  tag: "AMP-JWPLAYER"
+  tag: "AMP-KALTURA-PLAYER"
+  tag: "AMP-LIST"
+  tag: "AMP-LIST"
+  tag: "AMP-LIVE-LIST"
+  tag: "AMP-LIVE-LIST"
+  tag: "AMP-MATHML"
+  tag: "AMP-MOWPLAYER"
+  tag: "AMP-NEXXTV-PLAYER"
+  tag: "AMP-O2-PLAYER"
+  tag: "AMP-OOYALA-PLAYER"
+  tag: "AMP-PAN-ZOOM"
+  tag: "AMP-PINTEREST"
+  tag: "AMP-PLAYBUZZ"
+  tag: "AMP-POWR-PLAYER"
+  tag: "AMP-REACH-PLAYER"
+  tag: "AMP-REDDIT"
+  tag: "AMP-RIDDLE-QUIZ"
+  tag: "AMP-SOUNDCLOUD"
+  tag: "AMP-SPRINGBOARD-PLAYER"
+  tag: "AMP-TIMEAGO"
+  tag: "AMP-TWITTER"
+  tag: "AMP-VIDEO"
+  tag: "AMP-VIDEO-IFRAME"
+  tag: "AMP-VIMEO"
+  tag: "AMP-VINE"
+  tag: "AMP-VIQEO-PLAYER"
+  tag: "AMP-VK"
+  tag: "AMP-WISTIA-PLAYER"
+  tag: "AMP-YOTPO"
+  tag: "AMP-YOUTUBE"
+  tag: "ARTICLE"
+  tag: "ASIDE"
+  tag: "B"
+  tag: "BDI"
+  tag: "BDO"
+  tag: "BLOCKQUOTE"
+  tag: "BR"
+  tag: "BUTTON"
+  tag: "CAPTION"
+  tag: "CITE"
+  tag: "CODE"
+  tag: "COL"
+  tag: "COLGROUP"
+  tag: "DATA"
+  tag: "DD"
+  tag: "DEL"
+  tag: "DFN"
+  tag: "DIV"
+  tag: "DL"
+  tag: "DT"
+  tag: "EM"
+  tag: "FIGCAPTION"
+  tag: "FIGURE"
+  tag: "FOOTER"
+  tag: "H1"
+  tag: "H2"
+  tag: "H3"
+  tag: "H4"
+  tag: "H5"
+  tag: "H6"
+  tag: "HEADER"
+  tag: "HGROUP"
+  tag: "HR"
+  tag: "I"
+  tag: "INS"
+  tag: "KBD"
+  tag: "LI"
+  tag: "MAIN"
+  tag: "MARK"
+  tag: "NAV"
+  tag: "NOSCRIPT"
```

Comment:
Do you want noscript, since the parent tags will all just be `<span>`'s in this case? If so, you may want to include some of the noscript-only variant tags like `IMG`, `VIDEO`, `AUDIO`